icu4c/source/common/uresbund.cpp

```diff
@@ -2055,6 +2056,47 @@ void getAllItemsWithFallback(
     }
 }
 
+struct GetAllChildrenSink : public ResourceSink {
+    // Destination sink
+    ResourceSink& dest;
+
+    GetAllChildrenSink(ResourceSink& dest)
+        : dest(dest) {}
+    virtual ~GetAllChildrenSink() override;
+    virtual void put(const char *key, ResourceValue &value, UBool isRoot,
+           UErrorCode &errorCode) override {
+        ResourceTable itemsTable = value.getTable(errorCode);
+        if (U_FAILURE(errorCode)) { return; }
+        for (int32_t i = 0; i < itemsTable.getSize(); ++i) {
+            itemsTable.getKeyAndValue(i, key, value);
+            if (value.getType() == URES_ALIAS) {
+                ResourceDataValue& rdv = static_cast<ResourceDataValue&>(value);
+                StackUResourceBundle stackTempBundle;
+                UResourceBundle* aliasRB = getAliasTargetAsResourceBundle(rdv.getData(), rdv.getRes(), key, -1, rdv.getValidLocaleDataEntry(), nullptr, 0, stackTempBundle.getAlias(), &errorCode);
```

Comment:
```suggestion
                UResourceBundle* aliasRB = getAliasTargetAsResourceBundle(
                    rdv.getData(), rdv.getResource(), nullptr, -1, rdv.getValidLocaleDataEntry(), nullptr,
                    0, stackTempBundle.getAlias(), &errorCode);
```

1. line breaks
2. let's pass in key=nullptr since it does not make sense to give the real key when we don't have a containerResPath